docs/src/content/ignition/docs/guides/scripts.md

```diff
@@ -0,0 +1,241 @@
+# Advanced deployment with Hardhat scripts
```

Comment:
If we move to the advanced section maybe a name like "Deploying within Hardhat scripts"